packages/algob/src/internal/core/errors-list.ts

```diff
@@ -626,6 +627,23 @@ const accountErrors = {
 
 };
 
+const algorandNode = {
+  BAD_REQUEST: {
+    number: 410,
+    message: `status_code=%status%,
```

Comment:
Looks cryptic. Can we have a human readable error like in other places?

Reply:
it's in the next line